.circleci/soltest_all.sh

```diff
@@ -65,6 +65,14 @@ STEP=$(($STEP + 1))
 [[ " $RUN_STEPS " =~ " $STEP " ]] && EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V1=1 BOOST_TEST_ARGS="-t !smtCheckerTests" "${REPODIR}/.circleci/soltest.sh"
 STEP=$(($STEP + 1))
 
+# Run gas tests on semanticTests without optimizer.
```

Comment:
Why do we need two additional runs? Can't we just enforce the gas costs when we are running this combination of settings anyway?